specs/protocol/predeploys.md

```diff
@@ -70,6 +71,7 @@ or `Bedrock` or `Canyon`. Deprecated contracts should not be used.
 | L1FeeVault                    | 0x420000000000000000000000000000000000001a | Bedrock    | No         | Yes     |
 | SchemaRegistry                | 0x4200000000000000000000000000000000000020 | Bedrock    | No         | Yes     |
 | EAS                           | 0x4200000000000000000000000000000000000021 | Bedrock    | No         | Yes     |
+| RevenueSharer                 | 0x4200000000000000000000000000000000000024 | TBC        | No         | Yes     |
```

Comment:
Can we unpick that a bit -- do you mean (say) the OptimismWallet where the share of revenue will be sent? Or the actual formula for calculating revenue share?

My current plan is to have these either baked into the contract or configured via constructor args. Still configurable via proxy upgrade, of course.

Reply:
I'm more referencing this ongoing PR https://github.com/ethereum-optimism/specs-private/pull/3 that's attempting to define standard config for a standard chain. Wondering if this RevenueSharer needs to be captured in the standard config somehow.

Reply:
I think it may in future. I believe all chains in the registry will share revenue, so it might be a check on standard+frontier chains (is the revenuesharer deployed, does it point to an optimism wallet, does it send funds on L1 or L2).